generators/client/templates/src/main/webapp/app/blocks/handlers/_state.handler.js

```diff
@@ -62,15 +59,6 @@
                     stateChangeSuccess();
                 }
             });
-
-            function back() {
```

Comment:
Not anymore. Even before my PR, this was orphaned code. See how where it is still in use in 2.27.2:
- https://github.com/jhipster/generator-jhipster/blob/v2.x_maintenance/app/templates/src/main/webapp/scripts/app/_app.js#L64
- https://github.com/jhipster/generator-jhipster/blob/v2.x_maintenance/app/templates/src/main/webapp/scripts/app/account/login/_login.controller.js#L21

The intention for it seems to have been that if a previousState was set due to unauthorized access of a page, the login screen would be shown, followed by a "back()" to restore the previousState. But, there is nothing calling this "back()" function in 3.0 since it is being handled in the login callback.